doc/02-Installation.md

```diff
@@ -12,6 +12,26 @@
 
 The module ships with database functions for calculating the host and service availability in `etc/schema/`.
 
+### Grant Required Privileges
+
+Skip this step if you used the database configuration wizard during the icinga2 installation.
+
+Please proceed only if you did the setup manually as described here: 
+https://icinga.com/docs/icinga2/latest/doc/02-getting-started/#setting-up-the-mysql-database
+
+The import of the SQL functions will fail due to insufficient privileges.
+The required privileges are `CREATE, CREATE ROUTINE, ALTER ROUTINE`. 
+
+The following example assumes that your MySQL database is hosted on **localhost** and your user is named **icinga2**:
+
+```
+GRANT CREATE, CREATE ROUTINE, ALTER ROUTINE ON icinga2.* TO 'icinga2'@'localhost';
+```
+
+Please adapt the host and username to your environment.
+
+### Import Database Functions
```

Comment:
I'd like `Import Database Files` or something like that better because we're importing tables as well.

Reply:
Done and pushed with the squashed commit.